### ISSUE TITLE
Maximum call stack size exceeded on file upload

### ISSUE THREAD

**Issue by raphaelarias** [wearebraid/vue-formulate#385]
**Describe the bug**
Add the file input to a form. drag and drop. Nothing happens, the UI doesn't update, and the console has the error:
![image](https://user-images.githubusercontent.com/5008166/109428076-4f3e5200-79c3-11eb-947d-67c89d53c7be.png)

**To Reproduce**
I believe this is related to this error https://github.com/vuejs/vue/issues/7788 (even though it was closed and supposed to be fixed, I couldn't find any other explanation). I'm trying to create a way to reproduce. Currently, it happens in my Nuxt (2.15.2) project (perhaps because vuex is set as strict, but even if I try to disable it, still happens)

**Expected behavior**
It should display the image or file icon and show the progress bar.

**Screenshots**
![image](https://user-images.githubusercontent.com/5008166/109428233-2d919a80-79c4-11eb-854b-2a9ac1c307d4.png)

![image](https://user-images.githubusercontent.com/5008166/109428239-38e4c600-79c4-11eb-8977-f8d650ff61f6.png)

**Device information:**
 - Device: Mac
 - OS: MacOs Big Sur 11.2
 - Browser: Chrome (also happens on Firefox and Safari)
 - Version: 88

**Comment by justin-schroeder** [comment]
Happy to look into this, but it must have something to do with your implementation details. You mentioned Vuex? Are you somehow binding formulate data to vuex?

It would be really helpful if you could produce a reproduction on codepen or sandbox.

**Comment by raphaelarias** [comment]
No, not really. I believe it started happening when I updated Nuxt. I'm creation a repo attempting to reproduce and find the culprit. I keep you posted today :)

**Comment by raphaelarias** [comment]
There you go. Found it. I apologise if I created the ticket too early, but I deeply appreciate your prompt reply! I'll post below the problem in case someone has the same problem.

We've started testing a client-side logging library, I believe they may have implemented something in Vue that ended up breaking Vue Formulate (but nothing else, interestingly enough).

`import LogRocket from 'logrocket';` <- That was the problem. Removing fix it (I had added as a plugin to nuxt.config.js).
